app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
```

Comment:
More "pythonic" way of iteration:
```
for _ in argv:
    ...
```
Try it, it will greatly improve the readability of your code.

BTW, `_` is used if you do not want to use an iteration variable inside the cycle. In this case, you want to use it, so better to give it a descriptive name.